dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -201,3 +201,21 @@ CWPKI2043E.useraction=Ensure the the domain included in the value is a valid RDN
 CWPKI2044E=CWPKI2044E: The certificate is not an X.509 certificate. The certificate type is {0}.
 CWPKI2044E.explanation=The ACME service expects all certificates in the certificate chain to be X.509 certificates.
 CWPKI2044E.useraction=Ensure that all the certificates in the certificate chain are X.509 certificates and try again.
+
+# {1} is a time stamp.
+CWPKI2046I=CWPKI2046I: The certificate authority signed certificate with {0} serial number expires on {1}. The ACME service will request a new certificate from the ACME certificate authority at the {2} URI.
+CWPKI2046I.explanation=The ACME services starts requesting a new certificate based on the refreshBeforeExpiration property in the server configuration.
+CWPKI2046I.useraction=No action is required.
+
+# {1} is a time stamp.
+CWPKI2047W=CWPKI2047W: The certificate authority signed certificate with {0} serial number expired on {1}. The ACME service is not configured to automatically request a new certificate.
+CWPKI2047W.explanation=The certificate is expired and TLS/SSL requests will fail.
+CWPKI2047W.useraction=Use the ACME REST interface to request a new certificate. Or update the refreshBeforeExpiration property in the server configuration to a value greater than 1 day.
+
+CWPKI2048W=CWPKI2048W: The refreshBeforeExpiration property is set for {0} days which is longer than the duration of the certificate with {1} serial number. The duration of the certificate is {2} days. The refreshBeforeExpiration property is set to the default of {1} days.
+CWPKI2048W.explanation=The refresh time to request a new certificate is longer than the lifetime of the certificate. The refresh time is reset to the default.
+CWPKI2048W.useraction=Set the refreshBeforeExpiration property in the server configuration to an amount of time less than the duration of the certificate. Or remove the refreshBeforeExpiration property to use the default setting.
+
+CWPKI2049W=CWPKI2049W: The refreshBeforeExpiration property is set for {0} milliseconds which is too low. The refreshBeforeExpiration property is set to a minimum of {1}.
```

Comment:
Change,
"The refreshBeforeExpiration property is set for {0} milliseconds which is too low. The refreshBeforeExpiration property is set to a minimum of {1}."
to,
"The refreshBeforeExpiration property is set for {0} milliseconds which lower than the minimum of {1}."